.conf/dpv/protonvpn.template

```diff
@@ -26,12 +26,7 @@
 client
 dev tun
 proto $PROTOCOL
-
-remote $VPN_SERVER 443
-remote $VPN_SERVER 80
 remote $VPN_SERVER 1194
-remote $VPN_SERVER 5060
-remote $VPN_SERVER 4569
 
 remote-random
```

Comment:
You might want to remove this line too.

Reply:
Jep, not 100% sure on this, had the same idea, but I was just thinking that NordVPN might have a reason and at least it is only a noop in case 😄.